clang/test/CIR/Lowering/ThroughMLIR/bit.cir

```diff
@@ -0,0 +1,94 @@
+// RUN: cir-opt %s -cir-to-mlir -o %t.mlir
+// RUN: FileCheck %s --input-file %t.mlir
```

Comment:
In your next PRs can you please start writing these from source? You can use `-fno-clangir-direct-lowering`

Reply:
Ok, I'll rewrite bit.cir to bit.c